lib/ex_gram/updates/webhook.ex

```diff
@@ -0,0 +1,66 @@
+defmodule ExGram.Updates.Webhook do
+  @moduledoc """
+  Updates implementation that uses webhook method
+  """
+
+  use GenServer
+  require Logger
+
+  def update(token_hash, update) do
+    token_hash =
+      token_hash
+      |> String.to_atom()
+
+    GenServer.cast(token_hash, {:update, update})
+  end
+
+  def start_link({:bot, pid, :token, token}) do
+    token_hash =
+      token_hash(token)
+      |> String.to_atom()
+
+    GenServer.start_link(__MODULE__, {:ok, pid, token}, name: token_hash)
```

Comment:
I would set the name as `ExGram.Updates.Webhook.{token_hash}` (with `Module.concat/1`)